app/controllers/api/v1/users_controller.rb

```diff
@@ -27,6 +27,12 @@ def update
     end
   end
 
+  # DELETE /users/1
+  def destroy
+    set_user.destroy
+    head 204
```

Comment:
I think that if you are returning symbols in other endpoints you need to make the same here, use `head :no_content`